### PR TITLE
Create initial Jenkinsfile

### PR DESCRIPTION
**Pull request by adrianclay**
Have:

* Added RSpec
* Cleaned up the gems
* Added govuk-linter

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+version: '3.4'
+
+services:
+  app:
+    build: .
+    environment:
+      RACK_ENV: development
```
Comment (naliwajek) [inline docker-compose.yml]: Do we need `RAILS_ENV` or will Rails pick up `RACK_ENV` as well?

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,16 @@
+FROM ruby:2.5-alpine
+ARG BUNDLE_INSTALL_CMD
+ENV RACK_ENV=development
+
+WORKDIR /usr/src/app
+
+COPY Gemfile Gemfile.lock .ruby-version ./
+RUN apk --update --upgrade add build-base sqlite-dev tzdata && \
```
Comment (naliwajek) [inline Dockerfile]: Must remember to remove that `sqlite` dependency when we remove the gem 🐡

File: docker-compose.yml (hunk 1 above)
Comment (adrianclay) [reply docker-compose.yml]: Looks like Rails will pick it up:
https://github.com/rails/rails/blob/master/railties/lib/rails.rb#L73

**Comment by adrianclay** [comment]
Will merge for now.  Will address any further feedback in later PRs.

**Pull request merged**